crowbar_framework/config/locales/pacemaker/en.yml

```diff
@@ -123,6 +128,10 @@ en:
         drbd: 'Setting up DRBD requires a cluster of two nodes.'
         hae_repo: 'The HAE repositories have not been setup.'
         transport_value: 'Invalid transport value: %{transport}.'
+        ring_mode_value: 'Invalid ring mode: %{ring_mode}.'
+        second_ring_network_value: 'Network not found for setting up second corosync ring: %{second_ring_network}.'
```

Comment:
Nit: I prefer `Network "%{second_ring_network}" not found for setting up second corosync ring.'` - I think it's very slightly clearer.